run.sh

```diff
@@ -9,7 +9,13 @@ create)
 migrate_and_seed)
     echo "running migrate and seed"
     bundle exec rake db:migrate db:seed
+    RETURN_CODE=$?
+    while [ $RETURN_CODE -eq 1 ]; do
+      sleep 1
+      bundle exec rake db:migrate db:seed
+      RETURN_CODE=$?
+    done
     ;;
 esac
-bundle exec puma -b tcp://0.0.0.0:3000 -d -C config/puma_prod.rb
-tail -f /usr/src/app/log/logstash_production.json
+#bundle exec puma -b tcp://0.0.0.0:3000 -d -C config/puma_prod.rb
```

Comment:
Should't this script attempt to start the server after successfully running the migrations?